packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -727,6 +804,11 @@ export default class TableColumnResizeEditing extends Plugin {
 					ensureColumnResizerElement( viewWriter, item.item );
 				} );
 			}
+
+			if ( this._draggingState === 'afterdragging' ) {
```

Comment:
Relying on view `render` to change internal state sounds like a bad, bad idea 😕

We should be able to handle the `_isDragging` state using only drag-related events. As a fallback we could use some keyboard/mouse events to support that if that's impossible otherwise.